modules/images-imagestream-import-images-private-registry.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+// * assembly/openshift_images/managing-image-streams.adoc
+
+[id="images-imagestream-import-images-private-registry_{context}"]
+= Importing images from private registries
+
+An imagestream can be configured to import tag and image metadata from private image registries, requiring authentication.
+
+[NOTE]
+====
+When importing from an insecure registry, the registry URL defined in the Secret must include the :80 port suffix or the Secret is not used when attempting to import from the registry.
```

Comment:
It's not only for insecure registry. Both insecure and secure registry need add port if they defined one.
such as “upshift.mirror-registry.qe.devcluster.openshift.com:5000” and “upshift.mirror-registry.qe.devcluster.openshift.com:5001”， they are secure registries but could be configured with different authication.
@gabemontero  Could you help confirm this NOTE ?